cloudsync/cs.py

```diff
@@ -294,8 +286,6 @@ def do(self):
         import random  # pylint: disable=import-outside-toplevel
         mgrs = [*self.emgrs, self.smgr]
         random.shuffle(mgrs)
-        # conceptually, we should save the order of operations
-        # self.test_mgr_order.append(order_of(mgrs))
```

Comment:
I'm not sure this comment no longer applies

Reply:
not sure what this was supposed to mean, and the line it refers to is commented it out... is there something clearer we can put here?

Reply:
This is essentially a poorly worded todo. It relates to the idea that if we logged the order of operations at this point and the test failed, we could feed the order of operations back into a re-run of the test on a dev machine to reproduce the failure more easily

Reply:
ok, will clarify and put it back

Reply:
done.

Reply:
done.